configs/config.json.example

```diff
@@ -32,6 +32,8 @@
     "longer_eggs_first": true,
     "evolve_captured": false,
     "release_pokemon": true,
+    "avoid_circles": false,
+    "max_circle_size": 10,
```

Comment:
I like the additional context. What do you think about changing forts to pokestop in the configs to avoid confusion?
`'spin_pokestops': {
  'avoid_circles': false,
  max_circle_size: 10
}`